src/main/java/Storage.java

```diff
@@ -0,0 +1,132 @@
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Storage {
+
+    public static ArrayList<Task> data;
+    private static final Path fileLocation = Paths.get("./data", "duke.txt");
+    private static final Path directoryLocation = Paths.get("./data");
```

Comment:
Perhaps the variable names could be in upper cases, since they are final variables. It might be difficult for one to differentiate the constant variables.